src/commentscan.l

```diff
@@ -3991,7 +4006,14 @@ static bool handleToc(yyscan_t yyscanner,const QCString &, const StringVector &o
         }
         else
         {
-          level = level<=0 ? SectionType::MaxLevel : std::min(level,SectionType::MaxLevel);
+          // add: paule32 (testing: MSVC and GNU G++)
+#ifndef __GNUC__
+          if (level <= 0) level = SectionType::MaxLevel;
+          else            level = min(level, SectionType::MaxLevel);
```

Comment:
I don't understand which issue you have with `std::min`, but this is definitely not the way to solve it.